app/views/contexts/summary/group.py

```diff
@@ -151,6 +171,7 @@ def _build_blocks_and_links(
                         schema=schema,
                         location=location,
                         language=language,
+                        return_to="section-summary",
```

Comment:
Same as above.

Reply:
Removed this one and used instance attribute 👍